jsonnet/components/telemeter-client.libsonnet

```diff
@@ -75,4 +156,10 @@ function(params) {
       'ca-bundle.crt': '',
     },
   },
+
+  kubeRbacProxyMetricSecret: generateSecret.staticAuthSecret(cfg.namespace, cfg.commonLabels, 'telemeter-kube-rbac-proxy-metric') + {
```

Comment:
These static auth changes are being done in #1405 so you can revert this change?

Reply:
ok, I will revert this change and rebase when #1405 is merged.